Specs/Number/Italian/NumberModel.json

```diff
@@ -1,8 +1,177 @@
 [
   {
-    "Input": "192.",
```

Comment:
Please make the IP case equivalent to the one for French, add a "Comment" attribute explaining why, and re-enable it for dotnet.

Reply:
Working on both and will be pushed shortly.